src/media-groups.js

```diff
@@ -662,6 +662,20 @@ export const initialize = {
   }
 };
 
+const groupMatch = (list, media) => {
+  for (let i = 0; i < list.length; i++) {
+    if (playlistMatch(media, list[i])) {
+      return true;
+    }
+
+    if (list[i].playlists && groupMatch(list[i].playlists, media)) {
+      return true;
```

Comment:
should this be something like the following instead?
```suggestion
    if (list[i].playlists) {
      return groupMatch(list[i].playlists, media);
```

since, if we have a `playlists`, groupMatch is whatever recursing is.

Reply:
I don't think so because we want to consider other playlists as well. If we return the value of `groupMatch` here it might be false, while the next playlist in the list matches.

Reply:
ah, we don't want to return false here yet. I sort of missed the for loop. Cool, makes sense.